### PR TITLE
i18n: Add Chinese translation of exampleSite(docs)

### PR DESCRIPTION
**Pull request by naranyinyun**
Add Chinese translation of exampleSite.

Note: "batteries-included" have no explicit translation in Chinese, it means "自带电池", in this context it means "开箱即用(Out of the box)", but it also means "具有多种功能, 功能强大(Have multiple functions, Powerful function)".

There were some errors in the PR just now, now fixed(#11).

### CODE REVIEW

**Comment by imfing** [comment]
> And there are some errors that i don't know how to fix ＞﹏＜

Please post the errors here

**Comment by naranyinyun** [comment]
> > And there are some errors that i don't know how to fix ＞﹏＜
> 
> Please post the errors here

1. I don't know how to translate the main page
2. The card refer to ` {{< card link="../guide/organize-files.zh-cn" title="目录结构" icon="document-duplicate" >}}` but it displays 404, this error can be found in all cards, like this
image
3. The [blog](https://deploy-preview-12--hugo-hextra.netlify.app/zh-cn/blog/) displays 404

**Comment by imfing** [comment]
> > > And there are some errors that i don't know how to fix ＞﹏＜
> 
> > 
> 
> > Please post the errors here
> 
> 
> 
> 1. I don't know how to translate the main page
> 
> 2. The card refer to ` {{< card link="../guide/organize-files.zh-cn" title="目录结构" icon="document-duplicate" >}}` but it displays 404, this error can be found in all cards, like this
> 
> image
> 
> 3. The [blog](https://deploy-preview-12--hugo-hextra.netlify.app/zh-cn/blog/) displays 404
> 
> 

1. We don't need to do the home page for now as it's hard coded in the theme layouts "hextra-home". There are different ways to do this, ideally this page should get rendered from the _index.md which can be easily translated. 
2. Please remove the suffix `.zh-cn`
3. We need to translate the `blog/_index.md`

**Comment by naranyinyun** [comment]
> > > > And there are some errors that i don't know how to fix ＞﹏＜
> > 
> > > 
> > 
> > > Please post the errors here
> > 
> > 
> > 
> > 1. I don't know how to translate the main page
> > 
> > 2. The card refer to ` {{< card link="../guide/organize-files.zh-cn" title="目录结构" icon="document-duplicate" >}}` but it displays 404, this error can be found in all cards, like this
> > 
> > image
> > 
> > 3. The [blog](https://deploy-preview-12--hugo-hextra.netlify.app/zh-cn/blog/) displays 404
> > 
> > 
> 
> 1. We don't need to do the home page for now as it's hard coded in the theme layouts "hextra-home". There are different ways to do this, ideally this page should get rendered from the _index.md which can be easily translated. 
> 2. Please remove the suffix `.zh-cn`
> 3. We need to translate the `blog/_index.md`

OK,I will fix these

**Comment by naranyinyun** [comment]
All error has been fixed

**Comment by naranyinyun** [comment]
> > > > And there are some errors that i don't know how to fix ＞﹏＜
> > 
> > 
> > > 
> > 
> > 
> > > Please post the errors here
> > 
> > 
> > 
> > 1. I don't know how to translate the main page
> > 2. The card refer to ` {{< card link="../guide/organize-files.zh-cn" title="目录结构" icon="document-duplicate" >}}` but it displays 404, this error can be found in all cards, like this
> > 
> > image
> > 
> > 3. The [blog](https://deploy-preview-12--hugo-hextra.netlify.app/zh-cn/blog/) displays 404
> 
> 1. We don't need to do the home page for now as it's hard coded in the theme layouts "hextra-home". There are different ways to do this, ideally this page should get rendered from the _index.md which can be easily translated.
> 2. Please remove the suffix `.zh-cn`
> 3. We need to translate the `blog/_index.md`

All error has been fixed.

**Comment by naranyinyun** [comment]
Waiting for merge......

**Comment by imfing** [comment]
> Waiting for merge......

Can you address the comments I posted?

File: exampleSite/content/about/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "commentTranslate.source": "Google"
```
[the remark this answers is not in the document]
Comment (imfing) [reply exampleSite/content/about/.vscode/settings.json:2]: @naranyinyun

File: exampleSite/content/docs/getting-started.zh-cn.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+```shell
+$ hugo new content/_index.md
+$ hugo new content/docs/_index.md
+```
+
+### 在本地预览站点
+
+```shell
+$ hugo server
```
[the remark this answers is not in the document]
Comment (imfing) [reply exampleSite/content/docs/getting-started.zh-cn.md:68]: @naranyinyun

File: exampleSite/hugo.yaml [hunk 3]
```diff
@@ -18,9 +18,13 @@ languages:
     languageName: English
     weight: 1
     title: Hextra
+  zh-cn:
```
[the remark this answers is not in the document]
Comment (imfing) [reply exampleSite/hugo.yaml:21]: @naranyinyun

**Review by imfing — CHANGES_REQUESTED** [review]

**Comment by naranyinyun** [comment]
> > Waiting for merge......
> 
> Can you address the comments I posted?

Has been fixed

**Pull request merged**